proto/trust_zone/v1alpha1/trust_zone.proto

```diff
@@ -22,4 +23,5 @@ message TrustZone {
   repeated proto.federation.v1alpha1.Federation federations = 8;
   repeated proto.ap_binding.v1alpha1.APBinding attestation_policies = 9;
   optional string jwt_issuer = 10;
+  optional google.protobuf.Struct extra_helm_values = 12;
```

Comment:
Might this be `11`? Not sure it matters to much tbh.

Reply:
Yes, good catch. I think it was previously based on a different change that had another field.